src/main/java/microsoft/exchange/webservices/data/UserConfigurationDictionary.java

```diff
@@ -581,8 +583,7 @@ private Object constructObject(UserConfigurationDictionaryObjectType type,
     } else if (type.equals(UserConfigurationDictionaryObjectType.ByteArray)) {
       dictionaryObject = Base64EncoderStream.decode(value.get(0));
     } else if (type.equals(UserConfigurationDictionaryObjectType.DateTime)) {
-      Date dateTime = reader.getService()
-          .convertUniversalDateTimeStringToDate(value.get(0));
+      Date dateTime = new DateTimeParser().convertDateTimeStringToDate(value.get(0));
       if (dateTime != null) {
```

Comment:
What does dateTime == null mean? Doesn't the underlying class throw exceptions if the date is not valid?